shared/unattended/SLES-12.xml

```diff
@@ -554,23 +522,6 @@ echo ttyS0 >> /etc/securetty]]>
 chkconfig sshd on
 sed -i -e 's/\(PasswordAuthentication\s\)no/\1yes/g'  /etc/ssh/sshd_config
 service sshd restart
-#Since guest will perform system initialization at first boot after installation, so it's better to run a basic test to make following script to take effect.
```

Comment:
Yes, because the first nic interface is pinned to `eth0` by
```
+sed -i "s/^/#/g" /lib/udev/rules.d/75-persistent-net-generator.rules
```
But I'm sure if this is better than the original one.